src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -855,7 +855,7 @@ namespace FourPipeBeam {
             Real64 minFlow(0.0);
             Real64 maxFlowCool(0.0);
             minFlow = std::min(state.dataEnvrn->StdRhoAir * originalTermUnitSizeMaxVDot,
-                               state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesOAFlow * state.dataEnvrn->StdRhoAir);
+                               state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).MinOA * state.dataEnvrn->StdRhoAir);
```

Comment:
Hmm, I added this here to eliminate some diffs and within minutes https://github.com/NREL/EnergyPlus/pull/8951 was opened which makes the same change. Maybe I should remove these from here and let the diffs ride until that PR drops?